yarn.lock

```diff
@@ -6004,7 +6004,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"postcss@npm:^8.3.11, postcss@npm:^8.4.5":
+"postcss@npm:^8.3.11":
```

Comment:
the lockfile still has older versions of postcss. you need to de-dupe `postcss` so there is only one version